__tests__/app.test.js

```diff
@@ -0,0 +1,110 @@
+/**
+ * @jest-environment node
+ */
+
+const request = require('supertest');
+const nock = require('nock');
+const app = require('../src/app');
+const mockResponse = require('../src/mockResponse.js');
+
+const api = 'https://api.icndb.com';
+const mockError = { statusCode: 500, message: 'Error responding' };
+const errFunc = res => {
```

Comment:
What is this function doing? the name of it doesn't make me clear what your intended use is, so think about if you could rename it to make that more clear?

Reply:
This is the 'error' function. It is invoked wherever we need the error status and message to be sent. I wasn't sure of a proper name that I should call it so I just named it errFunc for errorFunction. What should I name it?

Reply:
const errFunc = res => {
```suggestion
const errFunc = res => {
  expect(res.statusCode).toEqual(500);
  expect(res.body.error).toEqual(mockError.message);
};
```
I put these statements in a function body to make them less repetitive.

Reply:
interesting refactor, perhaps you could name it `checkForError` `assertError` or something like that to describe the intent